config/console/console_init.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console
+spec:
+  selector:
+    matchLabels:
+      app: mcg-ms-console
+  template:
+    metadata:
+      labels:
+        app: mcg-ms-console
+    spec:
+      containers:
+        - name: mcg-ms-console
+          image: mcg-ms-console:latest
```

Comment:
can we give the image like this? it will check in local machine and dockerHub.

Reply:
What I see is, the path referred to fetch the image is built in `kustomization.yaml`. https://github.com/red-hat-storage/mcg-osd-deployer/blob/f465a00894ee87655f57f26309d7f0e29e6f53c2/config/console/kustomization.yaml#L13-L15

Reply:
@v-harihar yes correct